packages/uniforms-antd/package.json

```diff
@@ -32,8 +32,10 @@
     "src/*.tsx"
   ],
   "peerDependencies": {
-    "@ant-design/icons": "^4.0.0",
-    "antd": "^4.0.0",
+    "@ant-design/icons": "^5.0.0",
+    "antd": "^5.0.0",
+    "dayjs": "^1.11.13",
```

Comment:
I think we can consider keeping only `antd` and `react` in `peerDependencies` for now.